.eslintrc.js

```diff
@@ -77,6 +77,7 @@ module.exports = {
     'jsdoc/require-returns': ['error', { contexts: ['TSFunctionType'] }],
     'jsdoc/require-returns-type': ['error', { contexts: ['TSFunctionType'] }],
     'jsdoc/require-returns-description': ['error', { contexts: ['TSFunctionType'] }],
+    'react/no-unstable-nested-components': ['error', { allowAsProps: true }],
```

Comment:
To fix new eslint errors https://app.circleci.com/pipelines/github/mui/mui-x/26220/workflows/f133aad6-a5b3-405b-815d-86526f207584/jobs/150743

See https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md#rule-options for more details